### PR TITLE
SERVER-1204 metric improvements + added HTTP status code metrics

### PR DESCRIPTION
**Pull request by bolshoy**
- changed DefaultAgentGuid to DefaultAgentGUID as per golint recommendation
- in timed_metrics, added a real metrica realRate1Min - not aggregated one (keeping previous value)
- in counted_metrics, added support for http status metrics, also using real values

### CODE REVIEW

**Comment by bolshoy** [comment]
@rounds/server PR plz

**Comment by bolshoy** [comment]
:+1:

File: example/example.go [hunk 1]
```diff
@@ -24,7 +24,7 @@ var (
 	key             = flag.String("key", "", "private key pathname")
 	config          = flag.String("config", "", "pathname of JSON configuration file")
 	listen          = flag.String("listen", "127.0.0.1:8000", "listen address")
-	newrelicLicense = flag.String("newrelic-license", "", "Newrelic license")
+	newrelicLicense = flag.String("newrelic-license", "cf63166e7b192221fa0dbe2c3eb5b5409e273f95", "Newrelic license")
```
Comment (avivl) [inline example/example.go]: This is a public repo. Please take out our new relic key

File: agent.go [hunk 2]
```diff
@@ -43,23 +43,24 @@ type Agent struct {
...
 	return agent
 }
 
-//Run initialize Agent instance and start harvest go routine
+// Run initialize Agent instance and start harvest go routine
 func (agent *Agent) Run() error {
+
 	if agent.NewrelicLicense == "" {
-		return errors.New("please, pass a valid newrelic license key")
+		return errors.New("Cannot start NewRelic agent, pass a valid license key")
```
Comment (ofir-rounds) [inline agent.go]: Were is you'r manners? no Please?

**Comment by oryband** [comment]
@avivl @ofir-rounds this issue is still public in the pull requests page. I've contacted github asking them to remove this, but we should consider the newrelic key comprosied, and substitute it. In addition, the commit hash will eternally be public in github.

**Pull request merged**